.ci/appveyor.yml

```diff
@@ -20,6 +21,14 @@ cache:
   - "C:\\Users\\appveyor\\AppData\\Local\\coala-bears\\coala-bears"
   - "C:\\Users\\appveyor\\AppData\\Roaming\\nltk_data"
 
+# Workaround Appvayor bug:
+# Ignore YAMLLintBear
```

Comment:
Didn't test without the Ignore; should it work by default, or should someone enable the unbreakable-words?